README.md

```diff
@@ -35,23 +36,17 @@ ________
 
 ### What is Allstar?
 
-Allstar is a GitHub App installed on organizations or repositories to set and
-enforce security policies. Its goal is to be able to continuously monitor and
-detect any GitHub setting or repository file contents that may be risky or do
-not follow security best practices. If Allstar finds a repository to be out of
-compliance, it will take an action such as create an issue or restore security
-settings.
+Allstar is a GitHub App that continuously monitors GitHub organizations or repositories for adherence to security best practices. 
+If Allstar detects a security policy violation, it creates an issue to alert the repository or organization owner. 
+Future versions of Allstar will be able to automatically revert certain project settings to conform with the expected security best practice.
```

Comment:
This is here now. Maybe: "For a subset of settings, Allstar can automatically revert..." ?

Reply:
That's great! I'll make the change.